src/globals/core/_section-break.scss

```diff
@@ -44,6 +44,9 @@
 
   %govuk-section-break--visible {
     border-bottom: 1px solid $govuk-border-colour;
+    @include govuk-compatibility(govuk_elements) {
```

Comment:
What happens when elements overrides the invisible break?

Reply:
me being stupid and not moving it into where i said i was. fixed now.